docs/cpp/configure-intellisense-crosscompilation.md

```diff
@@ -0,0 +1,104 @@
+---
+Order: 9
+Area: cpp
+TOCTitle: Configure IntelliSense for cross-compiling
+ContentId: 381b7ce1-5766-49b0-ad26-f9eedae70e63
+PageTitle: Configure IntelliSense for cross-compilation
+DateApproved: 4/28/2020
+MetaDescription: Configure VS Code c_cpp_properties.json to get IntelliSense when you are compiling for a different platform
+---
+# Configure IntelliSense for cross-compiling
+
+When you use a compiler to target a different architecture than the one you are writing the code on, such as when your host machine is Windows but you are using an Arm compiler, you can configure the C/C++ Extension to provide proper IntelliSense.
+
+For IntelliSense to work, the C++ extension needs to be configured for the following:
+
+- The IntelliSense engine that emulates the target architecture/compiler so that data types reflect their correct sizes.
+- System and compiler defines need to be retrieved from the target compiler, or manually specified.
+- The path to library headers for the system and project need to be retrieved from the target compiler, or manually specified.
+
+Sometimes this information can be obtained automatically and sometimes it is provided by an extension. In all cases, you can enter it manually if needed. If you have trouble, feel free to file an issue in the [VS Code documentation repository](https://github.com/Microsoft/vscode-docs/issues).
+
+## Install target SDK
```

Comment:
Remove section about "Install target SDK". Example of what to install can be added in the "example of configuring IntelliSense for Linux ARM architecture.

Reply:
done